web/frontend/src/components/base.vue

```diff
@@ -244,8 +281,24 @@ const handleTabSelect = (tab: Tabs) => {
   }
 };
 
+const handleVisibilityChange = async () => {
+  if (document.visibilityState === 'hidden') {
+    pressed.clear();
+    eventQueue = [];
+    processing = false;
+    await stop();
+  }
+};
+
 onMounted(() => {
   initStreamState();
+  window.addEventListener('visibilitychange', handleVisibilityChange);
```

Comment:
Might as well stop anything that is going on if someone navigates away from the window. Just some extra fallbacks.